frontend/callback.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Authentication Callback</title>
+    <style>
+        #message {
+            color: #0a0707;
+            font-size: 1.2em;
+            margin: 20px;
+            padding: 10px;
+            background-color: #e2e8f0; 
+            text-align: center;
+        }
+    </style>
+</head>
+<body>
+    <div id="message">Loading...</div> <!-- Placeholder for messages -->
```

Comment:
Apparently we never use a div 😅. Maybe a <p> tag?

Reply:
eish alright 🤣

Reply:
We don't wanna pick fights here and there🤣